concent_api/core/message_handlers.py

```diff
@@ -369,6 +377,7 @@ def handle_send_force_subtask_results_response(request, client_message):
         state                     = Subtask.SubtaskState.REJECTED
         response_type             = PendingResponse.ResponseType.SubtaskResultsRejected
         provider_public_key       = client_message.subtask_results_rejected.report_computed_task.task_to_compute.provider_public_key
+        verification_deadline     = client_message.subtask_results_rejected.report_computed_task.task_to_compute.compute_task_def['deadline']
```

Comment:
Call this `subtask_deadline` or `computation_deadline`. To me `verification_deadline` sounds like the point in time at which we expect the requestor to be done with his verification and that's not true (you still need to add time windows to get that time).